core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineManagementClientImpl.java

```diff
@@ -149,7 +149,8 @@ public void apply(Iterable<OnlineCommand> commands) throws CommandFailedExceptio
     public ModelNodeResult execute(ModelNode operation) throws IOException {
         checkClosed();
         operation = adjustOperationForDomain.adjust(operation);
-        log.debugf("Executing operation %s", ModelNodeOperationToCliString.convert(operation));
+        log.debugf("Executing operation %s\nJSON format:\n%s", ModelNodeOperationToCliString.convert(operation),
+                operation.toJSONString(false));
```

Comment:
I believe we should be consistent with other `execute` methods, I suggest to do similar update mainly for `execute` on `Operation`. Not sure whether it would make sense to have it also for `execute` on `String` for some debugging purposes, I guess not.